docs/_docs/sql-reference/operational-commands.adoc

```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
 image::images/set-streaming.png[]
 
 
+== KILL QUERY
+
+The `KILL QUERY` command allows you to cancel a running query.
+
+[source,sql]
+----
+KILL QUERY {ASYNC} '<query_id>'
+----
+
+=== Description
+`ASYNC` is an optional parameter which returns control immediately without waiting for the cancellation to finish, and `query_id` can be retrieved via the `SQL_QUERIES` view.
+
+When a query cancelled via the `KILL` command, all parts of the query running on all other nodes are cancelled.
+
+This command available via JMX and CMD, also:
+
+- `QueryMXBean#cancelSQL`
```

Comment:
Could you please wrap this command example (and all others down below) up with the ascii [tabs] tag? Where the first tab shows the snipped for Unix systems (control.sh) while the second is dedicated for Windows users (control.bat)?

Here is an example:
https://github.com/apache/ignite/blame/IGNITE-7595/docs/_docs/includes/starting-node.adoc#L10

Reply:
Fixed.